### PR TITLE
Fix import error with scikit-image 0.17.0

### PR DESCRIPTION
**Pull request by ericpre**
There is an API changes in the last release of scikit-image:
https://github.com/scikit-image/scikit-image/blob/master/doc/release/release_0.17.rst#api-changes

### Progress of the PR
- [x] make import of `_upsampled_dft` compatible with scikit-image 0.17
- [x] ready for review.

### CODE REVIEW

File: hyperspy/_signals/signal2d.py [hunk 1]
```diff
@@ -22,7 +22,11 @@
 import dask.array as da
 import scipy as sp
 import logging
-from skimage.feature.register_translation import _upsampled_dft
+try:
+    # For scikit-image >= 0.17.0
+    from skimage.registration._phase_cross_correlation import _upsampled_dft
+except ModuleNotFoundError:
```
Comment (tjof2) [inline hyperspy/_signals/signal2d.py:28]: Why not `ImportError`?

Comment (ericpre) [reply hyperspy/_signals/signal2d.py:28]: This module doesn't exist in previous version.

Comment (tjof2) [reply hyperspy/_signals/signal2d.py:28]: More that `ModuleNotFoundError` is a sub-class of `ImportError` but not that it really matters.

Comment (tjof2) [reply hyperspy/_signals/signal2d.py:28]: Looks good to go!

**Review by tjof2 — APPROVED** [review]

**Pull request merged**